app.js

```diff
@@ -1,6 +1,27 @@
-function Book(title, author, pages, read) {
+
+let myLibrary = [];
+let operation = null;
+const formBtn = document.querySelector('#new-btn');
+const submitBook = document.querySelector('.submit-button');
+const form = document.querySelector('.input-form');
+
+// for the local storage
+let storedBooks = JSON.parse(localStorage.getItem("myLibrary"));
+if(storedBooks)
+  {
+    myLibrary = [...storedBooks];
+    render(myLibrary);
+  }
+
+function selectOperation(){
+  operation == null ? addBookToLibrary() : saveEditedInfo(operation);
+}
+
+function Book(id, title, author, description, pages, read) {
+    this.id = id;
     this.title = title;
     this.author = author;
+    this.description = description;
     this.pages = pages;
     this.read = read;
 
```

Comment:
On line 29 let's change the var for a let, to keep the scope of that variable within the function that's defining it.